File: src/main/java/ru/vaschenko/ParkPoint/dto/request/BookingRequestDto.java
package ru.vaschenko.ParkPoint.dto.request;

import ru.vaschenko.ParkPoint.dto.ParkingSpaceDto;

import java.time.LocalDateTime;
import java.util.List;

/**
 * создание бронирования
 */
public record BookingRequestDto(
        Long clientId,
        Long parkingSpaceId,
        LocalDateTime startTime,
        LocalDateTime endTime,
        Integer price
) {}

File: src/dto/BookingRequestDto.js
export class BookingRequestDto {
    constructor(clientId, parkingSpaceId, startTime, endTime) {
      this.clientId = clientId;
      this.parkingSpaceId = parkingSpaceId;
      this.startTime = new Date(startTime);
      this.endTime = new Date(endTime);
    }
  }